cirq/ops/linear_combinations.py

```diff
@@ -107,22 +107,22 @@ def __pow__(self, exponent: int) -> 'LinearCombinationOfGates':
         if self.num_qubits() != 1:
             return NotImplemented
         pauli_basis = {
-            common_gates.I,
+            I,
```

Comment:
Use `identity.I` as you have been elsewhere.

Reply:
Good find! Resolved. Thanks for the review.